src/services/sync/strategies/WebUntis/WebUntisBaseSyncer.js

```diff
@@ -0,0 +1,768 @@
+const { BadRequest } = require('@feathersjs/errors');
+const Syncer = require('../Syncer');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`../Syncer` import should occur after import of `cookie`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4061937)